api/interfaces.go

```diff
@@ -52,3 +52,8 @@ type SockchatWebsocketConnection interface {
 	ReadSocketMsg() (*SocketMessage, error)
 	ReadMsg() ([]byte, error)
 }
+
+// SockchatMetricsService generates reports based on user activity
+type SockchatMetricsService interface {
+	GetUserMessagesCounts(*UserActivityReportOptions) (*UserActivityReport, error)
```

Comment:
```suggestion
// SockchatReportsService generates reports based on user activity
type SockchatReportsService interface {
	GetUserActivityReport(*UserActivityReportOptions) (*UserActivityReport, error)
```
(optional) naming could be changed. `Metrics` usually refer to time series of system resources usage data, I would personally use `Reports` to distinguish user actions from application monitoring (also the comment above says `generates reports based on user activity` and not `generates metrics`).

Also `GetUserMessagesCounts` returns `UserActivityReport`, so I would just name it `GetUserActivityReport`

Reply:
done in [apply review suggestions](https://github.com/kacperf531/sockchat/pull/11/commits/4054b76b6854361e07ae3021db901a9e9bd32cb8)